plasma_cash/child_chain/__init__.py

```diff
@@ -1,5 +1,19 @@
 from flask import Flask
 
+callback = {}
+
+
+def on(event):
```

Comment:
Why we need another one instead of using the one in websocket.py?
Meanwhile, I am not really a fan of adding things in __init__.py. Is there any benefit or essentiality to put it here?

Reply:
1. `websocket.on` is designed to handle the event from clients, the global `on` is designed for pub/sub system in child_chain. It's possible to implement a more generalized pub/sub (`emit`/`on` function) system, but it will slightly complex then current one. I think either way will work.
2. It can be put in __init__.py or child_chain.py. If we are going to implement a generalized pub/sub system, __init__.py will be a better place for `on`/`emit`, or maybe in a separate file like pubsub.py.